### PR TITLE
Add error event args

### PR DESCRIPTION
**Pull request by nirinchev**

### CODE REVIEW

File: Platform.PCL/Realm.PCL/Extensions/CollectionNotificationsExtensionsPCL.cs [hunk 1]
```diff
@@ -18,12 +18,14 @@
```
Comment (fealebenpae) [inline Platform.PCL/Realm.PCL/Extensions/CollectionNotificationsExtensionsPCL.cs]: Why is this file in this PR?

Comment (nirinchev) [inline Platform.PCL/Realm.PCL/Extensions/CollectionNotificationsExtensionsPCL.cs]: Must have leaked from the cherry pick

File: Shared/Realm.Shared/Realm.cs [hunk 2]
```diff
@@ -212,7 +212,10 @@ private void NotifyChanged(EventArgs e)
             _realmChanged?.Invoke(this, e);
         }
 
-        public event ErrorEventHandler Error;
+        /// <summary>
+        /// Triggered when a Realm-level exception has coccurred.
```
Comment (fealebenpae) [inline Shared/Realm.Shared/Realm.cs]: Typo: `coccurred`

File: Platform.PCL/Realm.PCL/RealmPCL.cs [hunk 3]
```diff
@@ -95,6 +95,11 @@ public static Realm GetInstance(RealmConfiguration config = null)
         public event RealmChangedEventHandler RealmChanged;
 
         /// <summary>
+        /// Triggered when a Realm-level exception has coccurred.
```
Comment (fealebenpae) [inline Platform.PCL/Realm.PCL/RealmPCL.cs]: Typo: `coccurred`

**Pull request merged**